sdk/translation/Azure.AI.Translator.Document/src/Custom/MultipartRequestContent.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.AI.Translator.Document
+{
+    public partial class DocumentContent
+    {
+        /// <summary>
+        /// Document to be translated in the form
+        /// <para>
+        /// To assign a byte[] to this property use <see cref="BinaryData.FromBytes(byte[])"/>.
+        /// The byte[] will be serialized to a Base64 encoded string.
+        /// </para>
+        /// <para>
+        /// Examples:
+        /// <list type="bullet">
+        /// <item>
+        /// <term>BinaryData.FromBytes(new byte[] { 1, 2, 3 })</term>
+        /// <description>Creates a payload of "AQID".</description>
+        /// </item>
+        /// </list>
+        /// </para>
+        /// </summary>
+        internal BinaryData Document { get; }
+        /// <summary>
+        /// Glossary / translation memory will be used during translation in the form.
+        /// <para>
+        /// To assign a byte[] to the element of this property use <see cref="BinaryData.FromBytes(byte[])"/>.
+        /// The byte[] will be serialized to a Base64 encoded string.
+        /// </para>
+        /// <para>
+        /// Examples:
+        /// <list type="bullet">
+        /// <item>
+        /// <term>BinaryData.FromBytes(new byte[] { 1, 2, 3 })</term>
+        /// <description>Creates a payload of "AQID".</description>
+        /// </item>
+        /// </list>
+        /// </para>
+        /// </summary>
+        internal IList<BinaryData> Glossary { get; }
+
+        internal MultipartFormDataContent MultipartFormDataContent { get; }
```

Comment:
Hello please it is better not to add `MultipartFormDataContent` in the model. You can construct a multipartFormDataContent in the serialization method and convert it to RequestContent.